### DISCUSSION TITLE
Is there a way to load a filter just for one direction?

### DISCUSSION THREAD

**Discussion by larsimmisch** [baresip/baresip#2780] [General]
I'm working an a voice activity detector. My main usecase will be to run it on the rx side only, but it might make sense for tx only.

Is there a way to run a filter just as a decoder and not as an encoder using the config file?

**Comment by Tybil** [comment]
You can introduce your own filter specific setting, as far as i know there is no existing way to do it.

**Reply by larsimmisch** [reply]
Thanks, I'll do that then.
